pkg/logs/config/config.go

```diff
@@ -216,6 +218,8 @@ func NewLogsConfigKeys(configPrefix string) LogsConfigKeys {
 		AdditionalEndpoints:     configPrefix + "additional_endpoints",
 		BatchWait:               configPrefix + "batch_wait",
 		BatchMaxConcurrentSend:  configPrefix + "batch_max_concurrent_send",
+		BatchMaxSize:            configPrefix + "batch_max_size",
```

Comment:
Any reason BatchMaxSize is a config, but not available on passthroughPipelineDesc ?

Reply:
No good reason. I just excluded it initially because I had no reason to change from the default. Added now for consistency.